tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "__mocks__",
     "dist",
     "node_modules",
-    "**/__tests__/*"
+    "**/__tests__/*",
+    "Dangerfile.ts"
```

Comment:
Why the need to add this?

Reply:
> Why the need to add this?

Because otherwise the postbuild check, having **--no-dev**, will prompt _danger-plugin-digitalcitizenship_ as a missing dependency. This is because we add Dangerfile.ts to the build.
So I thought to exclude this file from the build, to avoid adding its dependency to package.json

Reply:
And that would be correct:`Dangerfile.ts` should not be included in the dist bundle, thus not processed by the `dependency-check` script

Reply:
I added this file to the exclude array of` io-fn-template` as well, it's ok?

Reply:
Ok, sorry I think I was wrong. So we were _already_ building `Dangerfile.ts` into the prod bundle, and you excluded it, right?

Perfect.